src/dbSta/src/dbSta.cc

```diff
@@ -188,6 +188,9 @@ class PathRenderer : public gui::Renderer
   void highlight(PathRef *path);
   virtual void drawObjects(gui::Painter& /* painter */) override;
 
+  // Force repaint to always occur
+  bool isRepaintRequired() override { return true; }
```

Comment:
I don't like the every debug Renderer has to do this (its a stumbling block for anyone writing a new one).  All the gui code outside of the src/gui has to trigger an update through Gui::redraw.  If that were mapped to fullRepaint then I think all these renders would be ok with the default caching implementation and wouldn't need to override it.

Reply:
I agree, its a little clumsy, I've removed it. But I noticed that 4 of the 9 renderers have no calls to redraw (dbSta, stt, grt, rsz), so they might behave oddly. If that's a problem, ideally they would be updated, but it would also be possible to either add a menu option to force a redraw or do a redraw when pause is asserted since the command line is disabled at that time?

Reply:
I think those four just draw a static result and registerRenderer calls redraw so they don't need to.  grt will change soon to using pause & redraw.